src/models/Project/createProject/index.js

```diff
@@ -61,14 +64,26 @@ const createProject = async ({
     return { errors }
   }
 
-  const project = await Project.query().insertAndFetch({
-    name: trimmedName,
-    slug,
-    description: trimmedDescription,
-    ownerId
-  })
+  const locale = await Locale.query().findById(defaultLocaleId)
```

Comment:
might be worth having validation:

```js
if (!locale) {
  throw new Error('This locale doesn’t exist')
}
```

Reply:
Right, planned to put this in but forgot about it 😓